src/cfnlint/core.py

```diff
@@ -215,15 +222,26 @@ def get_template_rules(filename, args):
     return(template, __CACHED_RULES, [])
 
 
-def run_checks(filename, template, rules, regions, mandatory_rules=None):
+def run_checks(filename, template, rules, regions, validate_registry_types=None, mandatory_rules=None):
```

Comment:
I think this should be last.  Just in case people are using positional arguments